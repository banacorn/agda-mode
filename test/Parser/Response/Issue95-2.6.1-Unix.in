(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue95 (/Users/omega/github/agda-mode/test/ManualTests/Issue95.agda).\n" t)
(agda2-highlight-load-and-delete-action "/var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-0")
(agda2-highlight-load-and-delete-action "/var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-1")
(agda2-highlight-load-and-delete-action "/var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-2")
(agda2-highlight-load-and-delete-action "/var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-3")
(agda2-highlight-load-and-delete-action "/var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-4")
(agda2-info-action "*Error*" "/Users/omega/github/agda-mode/test/ManualTests/Issue95.agda:8,5-9\nString != List Char\nwhen checking that the expression \"\\\\\" has type List Char" nil)
((last . 3) . (agda2-maybe-goto '("/Users/omega/github/agda-mode/test/ManualTests/Issue95.agda" . 144)))
(agda2-highlight-load-and-delete-action "/var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-5")
(agda2-status-action "")
Agda2> 