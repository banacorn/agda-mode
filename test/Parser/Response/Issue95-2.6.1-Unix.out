NoStatus
ClearRunningInfo
ClearHighlighting
RunningInfo 1 Checking Issue95 (/Users/omega/github/agda-mode/test/ManualTests/Issue95.agda).\n
HighlightingInfoIndirect /var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-0
HighlightingInfoIndirect /var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-1
HighlightingInfoIndirect /var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-2
HighlightingInfoIndirect /var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-3
HighlightingInfoIndirect /var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-4
DisplayInfo Error /Users/omega/github/agda-mode/test/ManualTests/Issue95.agda:8,5-9
String != List Char
when checking that the expression "\\" has type List Char
JumpToError /Users/omega/github/agda-mode/test/ManualTests/Issue95.agda 144
HighlightingInfoIndirect /var/folders/r2/lx4mzz2x1kn940m7lhnv0b_h0000gn/T/agda2-mode39869-5
NoStatus
